lib/Activity/Listener.php

```diff
@@ -175,6 +175,7 @@ public function generateCallActivity(Room $room, bool $endForEveryone = false, ?
 				->setTimestamp($this->timeFactory->getTime())
 				->setSubject('call', [
 					'room' => $room->getId(),
+					'avatar' => $room->getAvatar(),
```

Comment:
This should not be written into the database, but that is what you are doing here.

If you want to add it to the parameter (which I would **not** do), you actually have to do that here:
https://github.com/nextcloud/spreed/blob/659c2ba7ff5bad286f9ed3f456b5759b0906c8c6/lib/Activity/Provider/Base.php#L115-L121
But then you would also need to add it here to the docs:
https://github.com/nextcloud/server/blob/master/lib/public/RichObjectStrings/Definitions.php#L189-L194

---

But I think you should do the change here instead:
https://github.com/nextcloud/spreed/blob/659c2ba7ff5bad286f9ed3f456b5759b0906c8c6/lib/Activity/Provider/Base.php#L75-L79

Reply:
We don't store two formats of image. What we will do when need PNG and the image is SVG? We will drop support to SVG?

Reply:
Conversation avatars can not be SVG, so yeah the safety net can be removed and we just always return the PNG conversation avatar